soda/core/soda/execution/check/distribution_check.py

```diff
@@ -121,10 +121,20 @@ def sql_column_values_query(self, distribution_check_cfg, limit=1000000) -> str:
 
         partition_filter = self.partition.sql_partition_filter
         partition_str = ""
+
+        distribution_check_filter = distribution_check_cfg.filter
+
+        if distribution_check_filter:
```

Comment:
I would add a unit test to soda-core to test the `filter` on distribution checks. I will implement `sample` query to dist check and I want to make sure that my changes won't break anything. This part will mostly change in my future PR. However, I have the necessary info from this PR.